frontend/src/index.js

```diff
@@ -3,12 +3,16 @@ import ReactDOM from 'react-dom/client';
 import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import { ChakraProvider } from "@chakra-ui/react";
```

Comment:
Will using a 3rd party UI provider not make the website heavy due to more number of packages at the end?

Reply:
yes sir, it does make the initial rendering a bit late, i'll look into alternatives and change it in the next PR?